Dockerfile

```diff
@@ -1,110 +1,85 @@
-# See https://github.com/phusion/baseimage-docker/blob/master/Changelog.md
-# Based on Ubuntu 18.04 since v0.11
-FROM phusion/baseimage:0.11
+FROM aiidateam/aiida-core:latest
+
+LABEL maintainer="Materials Cloud Team <aiidalab@materialscloud.org>"
+
+
+# Note: The following config can be changed at build time:
+#   docker build  --build-arg NB_UID=200
+ARG NB_USER="scientist"
+ARG NB_UID="1000"
+ARG NB_GID="1000"
+
+ENV AIIDALAB_HOME /home/${SYSTEM_USER}
+ENV AIIDALAB_APPS ${AIIDALAB_HOME}/apps
 
 USER root
 
-# Add switch mirror to fix issue #9
-# https://github.com/aiidalab/aiidalab-docker-stack/issues/9
-RUN echo "deb http://mirror.switch.ch/ftp/mirror/ubuntu/ bionic main \ndeb-src http://mirror.switch.ch/ftp/mirror/ubuntu/ bionic main \n" >> /etc/apt/sources.list
-
-# install debian packages
-# Note: prefix all 'apt-get install' lines with 'apt-get update' to prevent failures in partial rebuilds
-RUN apt-get clean && rm -rf /var/lib/apt/lists/*
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-    tzdata 
-RUN apt-get update && apt-get install -y --no-install-recommends  \
-    bzip2                 \
-    build-essential       \
+RUN apt-get update && apt-get install -y  \
     ca-certificates       \
     cp2k                  \
     file                  \
-    git                   \
-    gnupg                 \
-    graphviz              \
-    locales               \
-    less                  \
     libssl-dev            \
     libffi-dev            \
-    psmisc                \
-    python-dev            \
-    python-pip            \
-    python-setuptools     \
-    python-wheel          \
-    python3-pip           \
-    python3-setuptools    \
-    python3-wheel         \
-    python-tk             \
     quantum-espresso      \
-    rsync                 \
-    ssh                   \
-    unzip                 \
-    vim                   \
-    wget                  \
-    zip                   \
   && rm -rf /var/lib/apt/lists/*
 
-# Add repo for postgres-9.6
-RUN echo "deb http://apt.postgresql.org/pub/repos/apt/ bionic-pgdg main" >> /etc/apt/sources.list.d/pgdg.list
-RUN wget -q https://www.postgresql.org/media/keys/ACCC4CF8.asc -O - | apt-key add -
-RUN apt-get update && apt-get install -y --no-install-recommends  \
-    postgresql-9.6        \
+# needed for jupyterlab
+RUN apt-get update && apt-get install -y \
+     nodejs                \
+     npm                   \
   && rm -rf /var/lib/apt/lists/*
 
-# fix locales
-RUN echo "en_US.UTF-8 UTF-8" > /etc/locale.gen && locale-gen
-ENV LC_ALL en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US.UTF-8
-
 
 # Quantum-Espresso Pseudo Potentials
 WORKDIR /opt/pseudos
 RUN base_url=http://archive.materialscloud.org/file/2018.0001/v2;  \
```

Comment:
```suggestion
RUN base_url=http://archive.materialscloud.org/file/2018.0001/v3;  \
```

Reply:
Please replace this by a download of the .aiida files, e.g.
https://archive.materialscloud.org/file/2018.0001/v3/SSSP_efficiency_pseudos.aiida

This way everyone will be using PPs with the same uuids.
See also 
https://github.com/marvel-nccr/ansible-role-aiida/blob/master/defaults/main.yml#L79
https://github.com/marvel-nccr/ansible-role-aiida/blob/master/tasks/aiida-pps-sssp.yml#L3

Reply:
done.